.env

```diff
@@ -1,4 +1,11 @@
 NODE_ENV=development
+<<<<<<< HEAD
```

Comment:
@Eirmas Never commit this file, even if it is draft

Reply:
Yeah these are conflict lines, wtf are they doing here? Hah